src/Analyzer/Passes/NormalizeCountVariantsPass.cpp

```diff
@@ -34,7 +34,7 @@ class NormalizeCountVariantsVisitor : public InDepthQueryTreeVisitorWithContext<
 
         auto & first_argument = function_node->getArguments().getNodes()[0];
         auto * first_argument_constant_node = first_argument->as<ConstantNode>();
-        if (!first_argument_constant_node)
+        if (!first_argument_constant_node || first_argument_constant_node->getResultType()->isNullable())
```

Comment:
I suspect it's better to check that the expression type is not changed before applying the optimization.
I think that's the reason why fuzzer found issues.

Reply:
Yep, make total sense, thanks, fixed.